core/src/main/java/io/seata/core/protocol/Version.java

```diff
@@ -69,8 +76,36 @@ public static String getChannelVersion(Channel c) {
      * @return the string
      * @throws IncompatibleVersionException the incompatible version exception
      */
-    public static String checkVersion(String version) throws IncompatibleVersionException {
-        // TODO: check
-        return version;
+    public static void checkVersion(String version) throws IncompatibleVersionException {
+        long current = convertVersion(CURRENT);
+        long clientVersion = convertVersion(version);
+        long divideVersion = convertVersion(VERSION_0_7_1);
+        if ((current > divideVersion && clientVersion < divideVersion) || (current < divideVersion && clientVersion > divideVersion)) {
```

Comment:
The verification function has not yet been reached and an exception has been thrown: io.netty.handler.codec.TooLongFrameException: Adjusted frame length exceeds 8388608: 2415945472-discarded